apps/route-checker/src/app/listener/pull.listener.ts

```diff
@@ -0,0 +1,27 @@
+import {logger, PubSubClient} from '@bike4life/commons'
+import {Message} from '@google-cloud/pubsub'
+import {pubSubSettings} from "../../../settings";
+import pullListenerService from "../services/pull.listener.service";
+
+export default async function startPullListener(): Promise<void> {
+    try {
+        const pubSubClient = new PubSubClient(pubSubSettings.projectId, logger)
+        const subscription = await pubSubClient.getSubscription(pubSubSettings.topicName, pubSubSettings.subscriptionName)
+
+        subscription.on('message', async (message: Message) => {
+            await pullListenerService.handleEvents(message.attributes?.type, JSON.parse(message.data.toString()))
```

Comment:
I would prefer if you avoid using the conditional chaining as this could lead to issues. Could you please check if the `attributes` property is defined, and otherwise log and error?